src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * duke.Parser class to parse the commands entered during run time
+ */
+
+public class Parser {
+
+    /**
+     * Splits the input line into description and time for the deadline
+     * or event
+     * @param string    the input line
+     * @return          array containing the description and time info
+     */
+    public static String[] splitDescriptionAndTime(String string) {
+        return string.split("/");
+    }
+
+    /**
+     * Extracts the description after splitting using splitDescriptionAndTime
+     * @param description_and_time  array after splitting the input line
+     * @return                      the description of the Task
```

Comment:
For javadoc comments, I believe there should be a spacing between your description of the method and the params, like:
```suggestion
     * Extracts the description after splitting using splitDescriptionAndTime
     *
     * @param description_and_time  array after splitting the input line
     * @return                      the description of the Task
```